src/language-css/parser-postcss.js

```diff
@@ -423,7 +423,11 @@ function parseNestedCSS(node) {
           };
         }
 
-        node.params = parseMediaQuery(params);
+        if (lowercasedName === "import") {
+          node.params = parseValue(params);
+        } else {
+          node.params = parseMediaQuery(params);
+        }
```

Comment:
Move this here https://github.com/ad1992/prettier/blob/e84c9949924e7b1046c07dff63a0c914abc0f6d7/src/language-css/parser-postcss.js#L394